Source/GUI/main.cpp

```diff
@@ -11,10 +11,6 @@
 
 
 #include <QtCore/QtPlugin>
-#if defined(_WIN32) && QT_VERSION >= 0x00050000 //Qt5
```

Comment:
Yes, I understand it causes an error for you because you're building dynamically. But my point is that you can't just remove code that might be required for other people that are building statically. I would suggest something like:

```
#if defined(_WIN32) && defined(QT_STATIC_BUILD) && QT_VERSION >= 0x00050000 //Qt5
    Q_IMPORT_PLUGIN(QWindowsIntegrationPlugin)
#endif
```